### PR TITLE
GZip note

### PR DESCRIPTION
**Pull request by JEuler**
Just little note for https://github.com/lejard-h/chopper/issues/120 issue

### CODE REVIEW

**Comment by JEuler** [comment]
Or maybe I should put it into FAQ

**Comment by JEuler** [comment]
Moved to the FAQ. @stewemetal what do you think? :)

**Comment by stewemetal** [comment]
There's that one sentence I'm not really OK with, but overall it's great!

Also, thanks for gathering these code samples and best practices in one place! Great idea! :)

File: converters/converters.md [hunk 1]
```diff
@@ -64,7 +64,24 @@ abstract class TodosListService extends ChopperService {
 
 }
 
-Response<T> convertResponse<T>(Response res) => 
+Response<T> convertResponse<T>(Response res) =>
     JsonConverter().convertResponse(res);
 ```
 
+## GZip converter example
+You can use converters for anything you want to modify the requests / responses.
```
[the remark this answers is not in the document]
Comment (JEuler) [reply converters/converters.md]: Yeah, looks nicer! Thx.

**Comment by JEuler** [comment]
I have fixed the comment, pls have a look :) @stewemetal @lejard-h

File: converters/converters.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JEuler) [reply converters/converters.md]: I'm resolving it due to I have fixed like you suggest :)

**Comment by JEuler** [comment]
@stewemetal All is okay? :)

**Comment by stewemetal** [comment]
Sorry, my my full time job got me distracted and forgot to approve. 😅 

All is OK!

**Comment by JEuler** [comment]
@stewemetal Chopper is in our heart :D Thanks)

**Pull request merged**